test/unit/resolveModule-test.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const { assert } = require('chai');
+
+const resolveModule = require('../../lib/resolveModule');
+
+
+describe('resolveModule()', () => {
+  const workingDirectory = path.join(__dirname, '..', 'fixtures');
+
+  it('resolves a local module name', () => {
+    assert.equal(
+      resolveModule(workingDirectory, 'requiredModule'),
+      path.join(workingDirectory, 'requiredModule')
+    );
+  });
+
+  it('resolves a local module name with .js extension', () => {
+    assert.equal(
+      resolveModule(workingDirectory, 'requiredModule.js'),
+      path.join(workingDirectory, 'requiredModule.js')
+    );
+  });
+
+  it('resolves an installed module name', () => {
+    assert.equal(
+      resolveModule(workingDirectory, 'coffeescript/register'),
+      'coffeescript/register'
```

Comment:
A minor thing, but maybe it's more future-proof to mock some generic module (i.e. `mocha`), the one that is unlikely to be removed as the library's dependency. I assume `coffeescript` is a legacy, so it's going to be removed in the future. It's nice not to hook up to legacy things in the tests.

Reply:
CoffeeScript is definitely a legacy as Dredd's direct dependency, but it is an existing JS transpiler (it's still developed and some people still use it). The `--require` option was introduced to mainly support transpilers, similar to `mocha --compilers js:babel-core/register`.

Any transpiler would do to verify the feature works, but `coffeescript` was already in dev dependencies from the past. Its objective advantage is it has zero dependencies itself, so using it to test the feature is very "cheap" (compared to e.g. `babel-core`, which has ~14 dependencies right now, and might require additional settings or packages to actually transpile something).

Also... it is a tiny little tribute to Dredd's origins... okay, with this one I admit I'm just making up things now 😄